lib/miq_expression.rb

```diff
@@ -96,13 +97,13 @@ def self._to_human(exp, options = {})
     when "between dates", "between times"
       col_name = exp[operator]["field"]
       col_type = get_col_type(col_name)
-      col_human, dumy = operands2humanvalue(exp[operator], options)
+      col_human, = operands2humanvalue(exp[operator], options)
```

Comment:
I know it's proper Ruby, but whenever people come across this and don't know Ruby can do this, they get confused.  I think it's more readable (and still rubocop compliant) to do:

```ruby
col_human, _ = operands2humanvalue(exp[operator], options)
```

or

```ruby
col_human, _val_human = operands2humanvalue(exp[operator], options)
```

(or whatever the right-hand-value is actually called)